src/AppBundle/Entity/User.php

```diff
@@ -78,9 +78,9 @@ class User extends BaseUser
     /**
      * @var bool
      *
-     * @ORM\Column(type="boolean")
+     * @ORM\Column(type="boolean", nullable=true)
```

Comment:
If you do change the database schema you have to make an migration using Docitrine migrations. `php app/console doctrine:migrations:diff`. But as I said, not here.